src/request-pipeline/destination-request/index.js

```diff
@@ -112,8 +112,11 @@ export default class DestinationRequest extends EventEmitter {
             this._send();
         }
 
-        else if (this._isDNSErr(err))
+        else if (this._isDNSErr(err)) {
+            this._abort();
```

Comment:
It makes sense to refactor whole code into method:
```js
_fatalError(msg) {
     this.aborted = true;
     this.req.abort();
     this.emit('fatalError', getText(msg, this.opts.url));
}
```

Remove `_abort` and use `_fatalError` everythere